package.json

```diff
@@ -62,9 +62,9 @@
     "lint": "eslint src test config --cache",
     "clear": "rimraf -rf ./dist && rimraf -rf ./gen",
     "build:es6node": "rimraf -rf dist/esnode && cross-env NODE_ENV=es6 babel src --out-dir dist/esnode && echo '{ \"type\": \"module\" }' > dist/esnode/package.json",
-    "build:es6browser": "rimraf -rf dist/esbrowser && cross-env NODE_ENV=es6 babel src --out-dir dist/esbrowser && grep -rl NodeMethod dist/esbrowser/ | xargs sed -i 's/.*NodeMethod.*//' && echo '{ \"type\": \"module\" }' > dist/esbrowser/package.json",
+    "build:es6browser": "rimraf -rf dist/esbrowser && cross-env NODE_ENV=es6 babel src --out-dir dist/esbrowser && grep -rl NodeMethod dist/esbrowser | xargs sed -i.bak '/.*NodeMethod.*/d' && echo '{ \"type\": \"module\" }' > dist/esbrowser/package.json",
```

Comment:
previous `sed` commands will not run on BSD-based distros (like Mac OS X). This should run on BSD and GNU, however I was only able to test on BSD